backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -129,6 +129,11 @@ def process_row_inner(self, sample, library, container, project, parent_containe
                                                                                            index=index_obj,
                                                                                            platform=platform_obj,
                                                                                            strandedness=library['strandedness'])
+
+        # Check if there's a sample with the same name
+        if Sample.objects.filter(name=sample['name']).exists():
```

Comment:
Another warning would be nice testing with iexact to see if there is another sample with the same name with different case. We could suggest checking the name.